style.md

```diff
@@ -1042,6 +1043,136 @@ func TestSigner(t *testing.T) {
 </td></tr>
 </tbody></table>
 
+### Embedding considered harmful
```

Comment:
The first line of this section - "Avoid embedding types in public structs" - is probably a better title. Let's use that instead.